apps/challenges/urls.py

```diff
@@ -25,4 +25,7 @@
         name='get_challenges_based_on_teams'),
     url(r'(?P<challenge_pk>[0-9]+)/challenge_phase_split$', views.challenge_phase_split_list,
         name='challenge_phase_split_list'),
+    url(r'challenge/challenge_host_team/(?P<challenge_host_team_pk>[0-9]+)/zip-upload/$',
```

Comment:
As far as I remember, we settled with using Underscores in the URLs and not hyphens, right?